actionpack/lib/action_dispatch/routing/redirection.rb

```diff
@@ -142,6 +142,8 @@ module Redirection
       # This will redirect the user, while ignoring certain parts of the request, including query string, etc.
       # <tt>/stories</tt>, <tt>/stories?foo=bar</tt>, etc all redirect to <tt>/posts</tt>.
       #
+      # The redirection happens as a <tt>301 Moved Permanently</tt> header unless otherwise specified using the <tt>:status</tt> option.
```

Comment:
What about including an example here too?

```suggestion
      # The redirect will use a <tt>301 Moved Permanently</tt> status code by
      # default. This can be overridden with the +:status+ option:
      #
      #   get "/stories" => redirect("/posts", status: 307)
```